gnome/gingerblue/Portfile

```diff
@@ -10,8 +12,13 @@ platforms           darwin
 license             GPL-3+
 maintainers         {gnome.org:ole @oleaamot} \
                     openmaintainer
-description         GNOME Gingerblue
-long_description    Locate Internet Radio Stations
+description         GNOME Gingerblue is Free Music Software for GTK+/GNOME.
+long_description    {*}${description} \
```

Comment:
Ole, note that the `long_description` should also start with the info from `description`.

And while that can be copy-pasted from the `description`, it's better to simply include it as I did here.

Reply:
Also, be careful not to use tabs anywhere. I fixed them for you already though, so this PR is all set.